src/features/pastelPaperWalletGenerator/PastelPaperWalletModal.tsx

```diff
@@ -19,6 +19,7 @@ type PastelPaperWalletModalProps = {
   info: {
     currencyName: string
   }
+  currentName: string
 }
 
 interface PDFDocumentProps {
```

Comment:
Can we use type instead of interface here?

Reply:
Updated